src/internet_identity/src/anchor_management/registration.rs

```diff
@@ -139,10 +160,22 @@ fn create_captcha<T: RngCore>(rng: T) -> (Base64, String) {
 
 // Check whether the CAPTCHA challenge was solved
 fn check_challenge(res: ChallengeAttempt) -> Result<(), ()> {
+    // avoid processing too many characters
+    if res.chars.len() > CAPTCHA_LENGTH {
+        return Err(());
+    }
```

Comment:
This is unrelated, right?

Reply:
I added it since we now do processing on the input where previously we did only an equality check.